packages/algob/src/lib/algo-operator.ts

```diff
@@ -60,4 +69,45 @@ export class AlgoOperatorImpl implements AlgoOperator {
       confirmedRound: txConfirmation[confirmedRound]
     };
   }
+
+  async deployASC (name: string, scParams: object, flags: ASCDeploymentFlags, payFlags: ASCPaymentFlags
+  ): Promise<ASCInfo> {
+    console.log("Deploying ASC:", name);
+    const result: ASCCache = await this.ensuredCompiled(name, false);
```

Comment:
Rename `ensuredCompiled` to `ensureCompiled`. This is a typo.

Reply:
ok